addon/components/frost-modal-input.js

```diff
@@ -11,6 +11,8 @@ export default Component.extend({
   $containerEl: null,
   $headerEl: null,
   $footerEl: null,
+  renderers: {},
+  validators: [],
```

Comment:
For completeness, should `showAllErrors` be initialized here as well to `false`?